flaml/automl.py

```diff
@@ -2287,6 +2289,17 @@ def custom_metric(
         }
         ```
 
+        cv_strategy: customized function, the strategy of conducting cross-validation. Default to average the optimization metric across folds.
+        We give an example here:
+
+        ```python
+        def cv_strategy(val_loss_folds):
+            return sum(val_loss_folds)/len(val_loss_folds)
```

Comment:
Please clarify the input and output.
My understanding is that the input is a list of 2-tuples. Each tuple has two elements, the first being a loss score to minimize and the second being a dict of all the metrics to log or None.
And the output is a 2-tuple with the same assumption similar to the "metric" function.

Please refer to the docstr of "metric".
Also, please add this argument in the docstr and the implementation of the constructor of `AutoML` similar to other kwargs.

Reply:
_"the second being a dict of all the metrics to log or None"_, What does this mean? Do we need to support customizing cv_score_agg_func for other metrics? I think in most cases, users only want to customize CV for one metric

Reply:
We allow users to define one metric to optimize for and multiple metrics to log in custom "metric" function (https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML#optimization-metric). So I wonder that we should allow the same in the custom "cv_score_agg_func".